src/Documenter.jl

```diff
@@ -547,13 +694,22 @@ function deploydocs(;
         return
     end
 
+    local cibranch, pull_request, repo_slug, tag, event_type, finaldeploy
+
+    if haskey(ENV, "CI")
+        cibranch, pull_request, repo_slug, tag, event_type, finaldeploy = read_ci_env(true, uploader=uploader)
+    else
+        cibranch = devbranch
+        pull_request = false
+        repo_slug = repo
+        tag = isempty(tag) && read_ci_env()[4]
+        event_type = "manual"
+        finaldeploy = haskey(ENV, "DOCUMENTER_KEY")
```

Comment:
When pushing locally, the user will probably have their normal SSH keys set up, so there's no need for `DOCUMENTER_KEY`. Also, `git_push` messes around with `~/.ssh/config`, which is fine on CI, but we might want to avoid that on user's computers.

Reply:
Done, it now checks for CI before overwriting - otherwise it just uses the user's config.